client/src/modules/multiple_payroll_indice/multiple_payroll_indice.js

```diff
@@ -84,7 +84,7 @@ function MultiplePayrollIndiceController(
 
     MultiplePayroll.openSearchModal(filtersSnapshot)
       .then((changes) => {
-        if (!changes.length) { return null; }
+        if (!changes || !changes.length) { return null; }
```

Comment:
Why is this one different from the others?

Reply:
There was a test for array `if (!changes.length)...` so I have just merge both if condition
```js
if (!changes) { return null; }
if (!changes.length) { return null; }
```
into 
```js
if (!changes || !changes.length) { return null; }
```
While we are talking about, maybe the previous code had an error and the good test is just `if (!changes) { return null; }`